docs/user-guide.md

```diff
@@ -164,6 +164,14 @@ INFO Create configuration ISO using: openshift-install agent create config-image
 INFO Download openshift-install from: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.12.10/openshift-install-linux.tar.gz
 ```
 
+### Rebuild
+
+Before rebuilding the appliance, e.g. for changing `diskSizeGB` or `ocpRelease`, use the `clean` command. This command removes the temp folder and prepares the `assets` folder for a rebuild. 
+Note: the command keeps the `cache` folder intact.
```

Comment:
```suggestion
Note: the command keeps the `assets/cache` folder intact.
```

Reply:
A minor proposal to make it clear where the cache is to be found.

Reply:
Sure, fixed.